src/Exceptions/HttpException.php

```diff
@@ -6,9 +6,10 @@
 use HttpSoft\Response\HtmlResponse;
 use HttpSoft\Response\JsonResponse;
 use Psr\Http\Message\ServerRequestInterface;
+use RuntimeException;
 
 abstract class HttpException extends \Exception {
-    public const CODE = 0;
+    public const CODE = 500;
```

Comment:
500 isn't a HTTP exception, 400 is. The 500 range is for server errors

Reply:
Yes but 4xx is for client related errors afaik

Reply:
HttpException to me suggests that there is an error with the request